### PR TITLE
Add missing semicolons

### PR DESCRIPTION
**Pull request by dstockwell**
And remove an unused file.

### CODE REVIEW

File: runtime/inner-PEC.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class StorageProxy {
 
   get() {
     return new Promise((resolve, reject) =>
-      this._port.HandleGet({ callback: r => {resolve(r)}, handle: this }));
+      this._port.HandleGet({ callback: r => {resolve(r);}, handle: this }));
```
Comment (smalls) [inline runtime/inner-PEC.js]: These inline semicolons seem weird.

File: runtime/test/manifest-test.js [hunk 2]
```diff
@@ -470,8 +470,8 @@ describe('manifest', function() {
         }
         ++count;
       }
-    })
-    assert.equal(count, 17);
+    });
+    assert.equal(count, 18);
```
Comment (smalls) [inline runtime/test/manifest-test.js]: Is this why travis is failing, maybe a bad merge or part of a different change?

File: runtime/webpack.config.js [hunk 3]
```diff
@@ -1,28 +0,0 @@
-/**
```
Comment (smalls) [inline runtime/webpack.config.js]: It might be clearer to do this in a separate commit (although I wouldn't unwind this cl to get it out).

**Comment by smalls** [comment]
lgtm although travis isn't happy (I left a comment near a suspect in the code).

File: runtime/test/manifest-test.js (hunk 2 above)
Comment (mmandlis) [reply runtime/test/manifest-test.js]: Yeah, i think this is a result of my "get rid of interleaved hack" pr that Doug just recently merged

Comment (dstockwell) [reply runtime/test/manifest-test.js]: Oh, oops! I messed up the rebase.

File: runtime/inner-PEC.js (hunk 1 above)
Comment (dstockwell) [reply runtime/inner-PEC.js]: Good catch, `r => {resolve(r);}` should really be `r => resolve(r)`

File: runtime/inner-PEC.js [hunk 4]
```diff
@@ -164,7 +164,7 @@ class InnerPEC {
         render(content) {
           this._pec._apiPort.Render({particle, slotName, content});
 
-          Object.keys(content).forEach(key => { this._requestedContentTypes.delete(key) });
+          Object.keys(content).forEach(key => { this._requestedContentTypes.delete(key); });
```
Comment (smalls) [inline runtime/inner-PEC.js]: Another arrow function that could use simplification (there are more below too).
